conanfile.py

```diff
@@ -3,7 +3,7 @@
 from conans import ConanFile
 
 
-class UVMConan(ConanFile):
+class UVWConan(ConanFile):
```

Comment:
I can say nothing about conan, sorry for this.

Reply:
Yes this should really be in another PR. 😊

Reply:
Seems small enough and reasonable enough (`UVM` vs `UVW` 😅). Not a problem to include it here. 👍

Reply:
Yes, see #247